pkg/apis/jvmbuildservice/v1alpha1/systemconfig_types.go

```diff
@@ -47,7 +47,7 @@ type SystemConfigList struct {
 
 const (
 	KonfluxGitDefinition          = "https://raw.githubusercontent.com/konflux-ci/build-definitions/refs/heads/main/task/git-clone/0.1/git-clone.yaml"
-	KonfluxPreBuildDefinitions    = "https://raw.githubusercontent.com/redhat-appstudio/jvm-build-service/main/deploy/tasks/pre-build.yaml"
+	KonfluxPreBuildDefinitions    = "https://raw.githubusercontent.com/rnc/jvm-build-service/PIPELINE/deploy/tasks/pre-build.yaml"
```

Comment:
Using `rnc` repo ?

Reply:
Task changes always have to be done in two stages - first with the reference to my branch then onces it merged into main it a subsequent PR to change the reference back. Its due to using the git resolver.